app/models/participant_profile/npq.rb

```diff
@@ -34,7 +34,7 @@ def participant_type
     end
 
     def fundable?
-      npq_application&.eligible_for_funding
+      npq_application&.eligible_for_dfe_funding
```

Comment:
should this change be in this PR if it's in  https://github.com/DFE-Digital/early-careers-framework/pull/2186 ?

Reply:
the commit is the same so don't think it will cause an issue